pytorch_lightning/logging/trains.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
this is not needed, we have here just loggers which existed in the time we were using `logging`

Reply:
Should I remove the file all together ?

Reply:
yes, pls